src/models/application.ts

```diff
@@ -1,13 +1,11 @@
-import { K8sGroupVersionKind, K8sModel } from './../dynamic-plugin-sdk';
+import { K8sModelCommon } from '@openshift/dynamic-plugin-sdk-utils';
+import { K8sGroupVersionKind } from './../dynamic-plugin-sdk';
 
-export const ApplicationModel: K8sModel = {
+export const ApplicationModel: K8sModelCommon = {
   apiGroup: 'appstudio.redhat.com',
   apiVersion: 'v1alpha1',
   kind: 'Application',
   plural: 'applications',
-  abbr: 'APP',
-  label: 'Application',
-  labelPlural: 'Applications',
```

Comment:
Hmm, I wonder if these are needed fields @rohitkrai03 if the Plugin adds them. These are half-holdovers from OpenShift Console to use models to show details on pages. I'm not a huge fan of this model, but the alternative is probably some mixture map to values.

Do you guys have a need for ResourceIcons? What about pages that read models and show labels like a page header?